src/index.test.ts

```diff
@@ -432,3 +432,142 @@ test("get all icon explicitly", async () => {
     }"
   `)
 })
+
+test("test collection alias", async () => {
+  const result = await postcss([
+    tailwindcss({
+      config: {
+        content: [
+          {
+            raw: '<span class="i-heroiconsstroke-archive-box"></span>',
+            extension: "html",
+          },
+        ],
+        plugins: [
+          iconsPlugin({
+            collections: getIconCollections("all"),
```

Comment:
It may not be necessary to use the full set of icons, which makes testing a bit slower